src/pages/AdminDashboard/AdminSettings.tsx

```
@@ -1,11 +1,322 @@
+   
+    // {
+    //   label: "Send feedback",
+    //   value: "sendfeedback",
+    
+    // },
+   
+   
+
+  ];
+
+  const [image, setImage] = useState("/default-profile.png"); 
```

ESLint

[error] 'image' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)